src/test/java/com/google/sps/TimingPropertiesTest.java

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps;
+ 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.sps.QuizTimingPropertiesUtils;
+ 
+@RunWith(JUnit4.class)
+public final class TimingPropertiesTest {
+  
+    private DatastoreService datastore;
+    private QuizTimingPropertiesUtils timing_properties_test;
+    public Entity user;
+    public Entity game;
```

Comment:
We'll want to make new entities in every test. Having them be global variables can cause issues with the tests. What if I set a property in the first test but that property conflicts with the expected output of the second test?

This also means that you should remove their declaration from the setUp() as well and just do it explicitly in each test function.

Reply:
The properties I set to these entitys are only set within tests so that they don't cross over, do you still think I should define them for each test? I kept them as global variables only to reduce the code

Reply:
We also would probably want to have DatastoreService datastore and QuizTimingPropertiesUtils timing_properties_test each be specific to each test. We don't want to share variables across multiple tests.

helper.setUp() should stay in the setUp() function, but everything else should be repeated on a per-test basis.